contracts/libraries/GnosisSafeStorage.sol

```diff
@@ -14,8 +14,8 @@ contract GnosisSafeStorage {
     uint256 internal threshold;
 
     // From /GnosisSafe.sol
-    bytes32 internal nonce;
-    bytes32 internal domainSeparator;
+    uint256 internal nonce;
+    bytes32 internal _deprecatedDomainSeparator;
```

Comment:
This is the only unresolved thing from previous PR:
> why do we use _ here for the variable name?
>
> I found it easier to understand/read the contracts when the names are 1:1 with the singleton contract. Also, In the > PR for GnosisSafeStorage tests, variable names are a part of the comparison.
>
> I'd understand _ as part of deprecation and yet another sign that one shouldn't use it

Reply:
:+1: